src/arviz_plots/plots/ppctstat.py

```diff
@@ -0,0 +1,214 @@
+"""ppc t-stat plot code."""
+from copy import copy
+from importlib import import_module
+
+from arviz_base import rcParams
+from arviz_base.labels import BaseLabeller
+
+from arviz_plots.plot_collection import PlotCollection
+from arviz_plots.plots.dist_plot import plot_dist
+from arviz_plots.plots.utils import process_group_variables_coords, set_figure_layout
+from arviz_plots.visuals import scatter_x
+
+
+def plot_ppc_tstat(
+    dt,
+    group="posterior_predictive",
+    t_stat="median",
+    var_names=None,
+    filter_vars=None,
+    sample_dims=None,
+    kind=None,
+    plot_collection=None,
+    coords=None,
+    backend=None,
+    data_pairs=None,
+    labeller=None,
+    aes_map=None,
+    plot_kwargs=None,
+    stats_kwargs=None,
+    pc_kwargs=None,
+):
+    """
+    Plot Bayesian t-stat for observed data and posterior/prior predictive.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data
+    group : str,
+        Group to be plotted. Defaults to "posterior_predictive".
+        It could also be "prior_predictive".
+    t_stat : str, float, default "median"
```

Comment:
We should also accept an arbitrary function. See example here https://python.arviz.org/en/stable/api/generated/arviz.plot_bpv.html